alias/docs/README.md

```diff
@@ -57,7 +57,7 @@ _Minimum_: <code>1</code>
 
 _Maximum_: <code>256</code>
 
-_Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
+_Update requires_: [Replacement](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
```

Comment:
This one (and others) should still be No interruption, only the specific ones I commented on should be Replacement.